src/wiseagents/agents/collaboration_wise_agents.py

```diff
@@ -428,7 +428,7 @@ def process_request(self, request: WiseAgentMessage):
 
         # Let the sender know that this agent has finished processing the request
         self.send_response(
-            WiseAgentMessage(message="", message_type=WiseAgentMessageType.ACK, sender=self.name, context_name=request.context_name,
+            WiseAgentMessage(message=llm_response.choices[0].message.content, message_type=WiseAgentMessageType.ACK, sender=self.name, context_name=request.context_name,
```

Comment:
This is meant to be just an ACK message that's why the messages was just empty. `llm_response.choices[0].message.content` will get added to the shared context instead.

Reply:
No it's to use it with the Assistant to chat in a standard chatbot experience. I know is just an ack, but does it hurt to have also the contents?

Reply:
Ok doesn't hurt to send the message contents.